### DISCUSSION TITLE
Using proxy with the blocked engines not all of them

### DISCUSSION THREAD

**Discussion by theonlyalm** [searxng/searxng#3292] [Q&A] [answered]
How to make the proxy works only with the engines that has been blocked me not all of them? To reduce the average number of requests sent from the proxy

**Comment by unixfox** [comment, marked as answer]
That's possible, it's not well documented but you can see here that you can specify the proxy for a specific engine: https://docs.searxng.org/admin/settings/settings_engine.html
